app/proc/processor_test.go

```diff
@@ -27,3 +28,24 @@ func TestSetDefault(t *testing.T) {
 
 	assert.EqualValues(t, expectedConf.System, p.Conf.System)
 }
+
+func TestFilter(t *testing.T) {
+	feedItem := feed.Item{Title: "Foo Bar (Part 1)"}
+	filter := Filter{Title: "(Part \\d+)"}
+	result, _ := filter.skip(feedItem)
```

Comment:
again, no error check even in the test. And also missing a test for the real error case

Reply:
3 tests you have here can be replaced with a single table test, like this one https://github.com/umputun/feed-master/blob/master/app/feed/parser_test.go#L33